prac_06/guitar_test.py

```diff
@@ -0,0 +1,13 @@
+from prac_06.Guitar import Guitar
+
+
+def main():
+    guitar1 = Guitar("Gibson L-5 CES", 1922, 16035.40)
+    guitar2 = Guitar("Another Guitar", 2013, 440)
+    print("{} get_age() - Expected 99. Got {}".format(guitar1.name, guitar1.get_age()))
```

Comment:
Lines 7 and 8 contain a math error in "_Expected_ {age}".

Reply:
I don't think it does, can you please elaborate?